src/briefcase/integrations/android_sdk.py

```diff
@@ -1366,27 +1370,37 @@ def start_emulator(
                     while not adb.has_booted():
                         if emulator_popen.poll() is not None:
                             raise BriefcaseCommandError(
-                                error_msg.format(
+                                failed_startup_error_msg.format(
                                     prologue="Android emulator was unable to boot!"
                                 )
                             )
 
                         # Try again in 2 seconds...
                         self.sleep(2)
         except BaseException as e:
-            # if the emulator exited, this should return its output immediately;
-            # if it is still running, this will quickly time out and print nothing.
-            with suppress(subprocess.TimeoutExpired):
+            self.tools.logger.info("Emulator output log", prefix=self.name)
+            try:
+                # if the emulator exited, this should return its output immediately
                 self.tools.logger.info(emulator_popen.communicate(timeout=1)[0])
+            except subprocess.TimeoutExpired:
+                self.tools.logger.info(
+                    "> Briefcase failed to retrieve emulator output "
```

Comment:
Is this chevron intentional? If it's intentional, I'm not sure what it's communicating.

Reply:
I went through a few iterations of what to print in this situation because initially, at least, I concluded that whatever is printed here could be construed as "the emulator failed to retrieve the logs". So, then I experimented with some bracket markup and then eventually just added "Briefcase" as the explicit actor that was trying to retrieve the logs.

At any rate, I can remove it.